ensime_shared/ensime.py

```diff
@@ -822,15 +826,11 @@ def detect_row_column_start():
 
 
 class EnsimeClientV1(ProtocolHandlerV1, EnsimeClient):
-    def __init__(self, *args, **kwargs):
```

Comment:
Have you tried it? This was my first approach but I had a "compile" error so I had to add the explicit calls due to the different amount of args in the `init` methods.